module/Application/src/Service/FileStorage.php

```diff
@@ -4,11 +4,8 @@
 
 class FileStorage
 {
-    private array $config;
-
-    public function __construct(array $config)
+    public function __construct(private array $config)
```

Comment:
This applies to a lot but not all changed properties, they can become `readonly`. It probably will not work with forms, but things like configs and services should be fine.